src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/Display.kt

```diff
@@ -177,7 +189,7 @@ abstract class Display {
             }
         }
 
-        this.height = i
+        this.height = i * 10
```

Comment:
Idk what this is but it doesn't belong in a commit titled "Fix spacing, change some fields to private"